Source/JavaScriptCore/heap/IsoSubspace.cpp

```diff
@@ -87,32 +87,28 @@ void IsoSubspace::didBeginSweepingToFreeList(MarkedBlock::Handle* block)
 
 void* IsoSubspace::tryAllocatePreciseOrLowerTierPrecise(size_t size)
```

Comment:
We can also remove preciseOnly mode code from IsoSubspace.

Reply:
Yeah, I was going to do that in a follow up.